src/util.js

```diff
@@ -58,7 +58,25 @@ export function chainWith(createDependentObservable) {
   });
 }
 
+/**
+ * @param {string} httpUrl  The Url
```

Comment:
just `url`, the protocol is customizable, maybe it's http, maybe it's data etc.

Reply:
I can't have the variable and the parameter with the same name, but I put urlParam instead of httpUrl, it's ok?